libraries/AP_NavEKF3/AP_NavEKF3.cpp

```diff
@@ -1239,6 +1239,11 @@ void NavEKF3::getAccelBias(int8_t instance, Vector3f &accelBias) const
     }
 }
 
+// returns active source set used by EKF3
+uint8_t NavEKF3::get_active_source_set(void){
```

Comment:
'cuz you'll be updating this file anyway, could you remove the "void" and add a little space between the round and curly brackets?  Sorry to be nitpicky.. it's not a blocker but if you could that'd be nice.

`uint8_t NavEKF3::get_active_source_set() {`